README.md

```diff
@@ -104,8 +104,12 @@ nuked.render({
           ],
         },
         size: {
-          radiusMin: 4,
-          radiusMax: 12,
+          sliderSingle: 10,
+          slider: [8, 22],
+          expression: { key: 'measureKey' },
+          autoRadiusValueRange: true,
+          radiusValueMin: 0,
+          radiusValueMax: 100
```

Comment:
I think we should give better property names for sn-map. Something like:
sliderSingle -> value
slider -> valueRange
radiusValueMin -> customMinRange
radiusValueMax -> customMaxRange